gokay/errors_test.go

```diff
@@ -1,72 +1,63 @@
-package gokay_test
+package gokay
```

Comment:
This pattern is often used to black-box test code, as the tests can no longer use unexported symbols. However, that's what behavioural testing is for, imo.